src/dayNightSystem/dayNightClock.mjs

```diff
@@ -6,69 +6,101 @@
 // note: don't run more than one day/night clock at a time.  doing so will cause multiple
 //       filters to be applied to the screen, which won't look too nice. :o)
 
-import { Prim, Thread } from 'sphere-runtime';
+import { Thread } from 'sphere-runtime';
 
 import InGameTime from './inGameTime';
 
-const DayMask = Color.Transparent,
-      TwilightMask = new Color(0.5, 0.125, 0.0625, 0.625),
-      NightMask = new Color(0.0, 0.0, 0.125, 0.5625);
+const DayMask = [0.0, 0.0, 0.0, 0.0],
+      TwilightMask = [0.5, 0.125, 0.0625, 0.625],
+      NightMask = [0.0, 0.0, 0.125, 0.5625];
+
+const Day = 0,
+      Night = 1,
+      Fading = 2;
 
 export default
 class DayNightClock extends Thread
 {
 	constructor()
 	{
 		super({ priority: 1 });
+		this.inGameTime = new InGameTime(0, 0, 0);
 
 		console.log("initializing day/night clock", `time: ${this.now()}`);
-		this.currentMask = Color.Transparent;
+		let white = Color.White;
```

Comment:
This line could be removed entirely: White is the default color for vertices if no color is specified, exactly so that you can make colorizable "silhouettes" like you're doing here.

Reply:
removed